tests/test_basic.py

```diff
@@ -2,7 +2,50 @@
 import unittest
 
 from skoolkittest import SkoolKitTestCase
-from skoolkit.basic import BasicLister
+from skoolkit.basic import get_number, TextReader, BasicLister, VariableLister
+
+class get_number_Test(SkoolKitTestCase):
```

Comment:
I'm thinking this test class should go the same way as TextReaderTest. Is anything being tested here that isn't already tested by BasicListerTest or VariableListerTest? Really, ``get_number()`` is a private function (and so should be renamed ``_get_number()``).